src/Web/Sfa.Das.Sas.ApplicationServices/Handlers/SitemapHandler.cs

```diff
@@ -54,11 +55,39 @@ public SitemapResponse Handle(SitemapQuery message)
             };
         }
 
+        private IEnumerable<string> GetFrameworkDetailsInSeoFormat()
+        {
+            var frameworks = _getFrameworks.GetAllFrameworks();
+
+            return this.BuildFrameworkSitemap(frameworks);
```

Comment:
this. not needed (and it line 69)

Reply:
I see the dotSettings suggests removing that so have taken them out.